scripts/monorepo/getAllPackageInfo.js

```diff
@@ -0,0 +1,25 @@
+const { spawnSync } = require('child_process');
+const { readConfig } = require('../read-config');
+const path = require('path');
+const findGitRoot = require('./findGitRoot');
+
+module.exports = function getAllPackageInfo() {
+  const gitRoot = findGitRoot();
+  const results = spawnSync('git', ['ls-tree', '-r', '--name-only', '--full-tree', 'HEAD']);
```

Comment:
I think it's marginally faster to run `git ls-files '*/package.json'`, but has the benefit of not needing to filter out entries after retrieving them.

Reply:
oh cool tip!

Reply:
i'll fix it next go around maybe :)